python/cudf/cudf/core/dataframe.py

```diff
@@ -6705,6 +6698,18 @@ def append(
         2  2
         3  3
         4  4
+
+        .. pandas-compat::
+            **DataFrame.append**
+
+            If a list of dict/series is passed and the keys are all contained in
+            the DataFrame's index, the order of the columns in the resulting
+            DataFrame will be unchanged.
+            Iteratively appending rows to a cudf DataFrame can be more
+            computationally intensive than a single concatenate. A better
+            solution is to append those rows to a list and then concatenate
+            the list with the original DataFrame all at once.
+            `verify_integrity` parameter is not supported yet.
```

Comment:
This is not a pandas compatibility note (in fact, the same note appears in the pandas docstring for this method), so can we restore to the previous state please?